src/stores/firebaseAuthStore.ts

```diff
@@ -90,7 +90,17 @@ export const useFirebaseAuthStore = defineStore('firebaseAuth', () => {
 
   const getIdToken = async (): Promise<string | null> => {
     if (currentUser.value) {
-      return currentUser.value.getIdToken()
+      try {
+        return await currentUser.value.getIdToken()
+      } catch (error: any) {
```

Comment:
Errors should usually be typed `unknown`.
https://www.typescriptlang.org/play/?ts=4.0.2#example/unknown-in-catch